ui/widget.js

```diff
@@ -368,7 +368,7 @@ $.Widget.prototype = {
 			.removeAttr( "aria-disabled" );
 
 		// Clean up events and states
-		this.bindings.off( this.eventNamespace );
+		$( Array.from( this.bindings ) ).off( this.eventNamespace );
```

Comment:
I'm not that worried about this since it's in `destroy`. That shouldn't happen too often.